client/src/views/CreateCopyrightApplication.vue

```diff
@@ -230,16 +230,11 @@
                     type="number"
                     v-model="form.authorYearOfBirth"
                     :disabled="sending"
-                    required
                     @blur="validateField('authorYearOfBirth')"
```

Comment:
@blur="validateField" is not needed since this is not using "custom" validations